test/integration/api.bats

```diff
@@ -68,9 +68,68 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker images" {
-	skip
+	start_docker 3
+	swarm_manage
+
+	# no image exist
+	run docker_swarm images 
```

Comment:
I think here you could run `images -q` and simply check that the output has 0 lines